### ISSUE TITLE
Cli-data-forwarder frequency off by one

### ISSUE THREAD

**Issue by luisomoreau** [edgeimpulse/edge-impulse-cli#16]
Issue from a forum thread: https://forum.edgeimpulse.com/t/cli-data-forwarder-frequency-off-by-one/2293 

I just had a look at the source code here: https://github.com/edgeimpulse/edge-impulse-cli/blob/840c0ead1d3f94bbf152210196eb01de859ca376/cli/data-forwarder.ts#L590

I do not see anything that could increase the frequency by 1 except maybe the let l = lines[1];

**Comment by sillevl** [comment]
I made a pull request that solves this problem: https://github.com/edgeimpulse/edge-impulse-cli/pull/15

The problem lies within the `String.prototype.split()` method that is used. If there are no more characters after the last `\n` delimiter, it will result in an empty element at the end of the returned array (which is almost always the case). The last element of the array thus cannot contain a valid 'line'. This element should thus be ignored when counting the 'lines' for the given time. I described this in my PR https://github.com/edgeimpulse/edge-impulse-cli/pull/15

https://github.com/edgeimpulse/edge-impulse-cli/blob/840c0ead1d3f94bbf152210196eb01de859ca376/cli/data-forwarder.ts#L588

@luisomoreau `let l = lines[1];` is only used to inspect the data format, so it does not have any impact on the data frequency calculation.
